scripts/openstack-quickstart-demosetup

```diff
@@ -66,6 +66,10 @@ if [ "$DB" = "postgresql" ] ; then
             sed -i 's/\s*sameuser$//' $DATADIR/pg_hba.conf # adapt config syntax to postgresql-9
         fi
     fi
+    # increase max connections for multinode setup (this needs a restart)
+    sed -i "s/^max_connections=.*/max_connections=1000/g" $DATADIR/postgresql.conf
+    start_and_enable_service $DB
+    stop_and_disable_service $DB
```

Comment:
stopand disable is not right.. can you put the sed in front of the first postgresql start?